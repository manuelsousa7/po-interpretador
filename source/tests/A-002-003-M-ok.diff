*** tests/A-002-003-M-ok.out	Wed Nov  9 22:45:10 2016
--- tests/A-002-003-M-ok.outhyp	Wed Nov 16 13:35:30 2016
***************
*** 7,37 ****
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Nome do program: Menu de Edição
! 1 - Listar programa
! 2 - Executar programa
! 3 - Adicionar expressão ao programa
! 4 - Substituir expressão no programa
! 5 - Mostrar os identificadores presentes no programa
! 6 - Mostrar os identificadores do programa sem inicialização explícita
! 0 - Sair
! Escolha uma opção: (eq 1 1)
! (ne 1 2)
! (lt 1 2)
! (gt 1 2)
! (le 1 2)
! (ge 1 2)
! (not 1)
! (and 1 0)
! (or 0 aa)
! Menu de Edição
! 1 - Listar programa
! 2 - Executar programa
! 3 - Adicionar expressão ao programa
! 4 - Substituir expressão no programa
! 5 - Mostrar os identificadores presentes no programa
! 6 - Mostrar os identificadores do programa sem inicialização explícita
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Criar
  2 - Abrir
--- 7,22 ----
  6 - Escrever Programa
  7 - Manipular Programa
  0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Criar
! 2 - Abrir
! 3 - Guardar
! 4 - Criar Programa
! 5 - Ler Programa
! 6 - Escrever Programa
! 7 - Manipular Programa
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Criar
  2 - Abrir
