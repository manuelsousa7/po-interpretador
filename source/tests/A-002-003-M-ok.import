(eq 1 1)
(ne 1 2)
(lt 1 2)
(gt 1 2)
(le 1 2)
(ge 1 2)
(not 1)
(and 1 0)
(or 0 aa)
