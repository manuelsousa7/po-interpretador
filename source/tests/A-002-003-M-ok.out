Menu Principal
1 - Criar
2 - Abrir
3 - Guardar
4 - Criar Programa
5 - Ler Programa
6 - Escrever Programa
7 - Manipular Programa
0 - Sair
Escolha uma opção: Nome do program: Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: (eq 1 1)
(ne 1 2)
(lt 1 2)
(gt 1 2)
(le 1 2)
(ge 1 2)
(not 1)
(and 1 0)
(or 0 aa)
Menu de Edição
1 - Listar programa
2 - Executar programa
3 - Adicionar expressão ao programa
4 - Substituir expressão no programa
5 - Mostrar os identificadores presentes no programa
6 - Mostrar os identificadores do programa sem inicialização explícita
0 - Sair
Escolha uma opção: Menu Principal
1 - Criar
2 - Abrir
3 - Guardar
4 - Criar Programa
5 - Ler Programa
6 - Escrever Programa
7 - Manipular Programa
0 - Sair
Escolha uma opção: 